save-cloud-common/src/commonMain/kotlin/org/cqfn/save/entities/Organization.kt

```diff
@@ -0,0 +1,28 @@
+package org.cqfn.save.entities
+
+import org.cqfn.save.utils.LocalDateTime
+
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+
+/**
+ * @property name organization
+ * @property ownerId organization
+ * @property dateCreated date created organization
+ */
+@Entity
+@Serializable
+@Suppress("USE_DATA_CLASS")
+data class Organization(
```

Comment:
Then we will have to manually redefine the equals() method, because in the tests there will be a comparison by reference, and not by class fields.

Reply:
Ok, but we absolutely shouldn't put something into actual production logic just for tests to pass.

Reply:
> You shouldn't be using data class here, because generated `equals` interact poorly with Hibernate

Actually, I was not quite right: since we exclude auto-generated ID from `equals` (by moving it away from the primary constructor of data class), it won't cause problems. And in this particular case, when there are no associations with other entities, data class won't cause any problems.

Here is the best explanation: https://www.jpa-buddy.com/blog/best-practices-and-common-pitfalls, especially 'Using Data Classes for JPA Entities' section.